### DISCUSSION TITLE
Does pyshark pyshark support TLS decrypt with SSL key file ?

### DISCUSSION THREAD

**Discussion by RaywayJohn** [KimiNewt/pyshark#510] [General]
Hi Guys ,
Hope you're doing great .
May I know if  pyshark supports TLS decrypt with SSL key file ?

tshark command like this :
tshark -Vnxr capFile -F Fileformat -R "Display_Filter" -o tls.keylog_file:/path/to/keylog.file > textfile

https://www.comparitech.com/net-admin/decrypt-ssl-with-wireshark/
https://tshark.dev/export/export_tls/

**Comment by bennettbrad** [comment]
Yes, it goes something like this:
parameters_dict = {'-o' : "ssl.keylog_file:/home/user/ssl_keys"}
filter = '''json''
cap_json = pyshark.FileCapture(file_name, display_filter=filter, custom_parameters=parameters_dict)

**Reply by RaywayJohn** [reply]
Hi @bennettbrad ,
Thanks for the confirmation and provide a code sample . Thank you very much
